### ISSUE TITLE
SSD Training 

### ISSUE THREAD

**Issue by Maithilishetty** [bosch-ros-pkg/bstld#19]
Hi, I tried using the SSD config file to train. However, my training loss is almost constant at 0.346 with continuous spikes in between. After nearly ~150 steps, my loss continuous increases. I have tried tuning the parameters but no change in results unfortunately. My learning rate is also quite low. Is there anything I'm missing or should be looking into? 

Thanks!

**Comment by karstenBehrendt** [comment]
Hi @Maithilishetty, sorry for the late reply. Frameworks change all the time and there is a chance that even with fixed library versions, trainings do not perform as expected.

Please understand that I cannot debug and support training runs on the dataset. If you find a way to fix this, feel free to provide a pull request.
